server/pulp/server/webservices/middleware/postponed.py

```diff
@@ -31,39 +18,12 @@ def _get_operation_postponed_body(exception):
         report = TaskResult.from_async_result(exception.call_report)
     serialized_call_report = report.serialize()
     for task in serialized_call_report['spawned_tasks']:
-        href_obj = serialization.dispatch.task_result_href(task)
+        href_obj = serializers.dispatch.task_result_href(task)
         task.update(href_obj)
 
     return json.dumps(serialized_call_report, default=json_util.default)
 
 
-class PostponedOperationMiddleware(object):
-    """
-    Catch OperationPostponed exceptions and return an HTTP Accepted response
-    along with the proper serialization of the asynchronous call information.
-    """
-
-    def __init__(self, app):
-        self.app = app
-        self.headers = {'Content-Encoding': 'utf-8',
-                        'Content-Type': 'application/json',
-                        'Content-Length': '-1'}
-
-    def __call__(self, environ, start_response):
-
-        try:
-            return self.app(environ, start_response)
-
-        except OperationPostponed, e:
-            body = _get_operation_postponed_body(e)
-
-            self.headers['Content-Length'] = str(len(body))
-            start_str = '%d %s' % (e.http_status_code, http_responses[e.http_status_code])
-
-            start_response(start_str, [(k, v) for k, v in self.headers.items()])
-            return [body]
-
-
 class DjangoPostponedOperationMiddleware(object):
```

Comment:
Can this be renamed from `DjangoPostponedOperationMiddleware` to `PostponedOperationMiddleware`. You'll have to make some updates to match.